crates/editor/src/editor.rs

```diff
@@ -3969,6 +3973,39 @@ impl Editor {
         }
     }
 
+    fn accept_partial_copilot_suggestion(
+        &mut self,
+        _: &AcceptPartialCopilotSuggestion,
```

Comment:
We have a set of tests on copilot already: https://github.com/zed-industries/zed/blob/e6bec9a24a12773fc8e9d4ed0a4d37cb06bed75c/crates/editor/src/editor_tests.rs#L7392

That test function is quite large, but we should be able to create a new one with the same LSP set-up, testing the new code, let's do that.